tests/Comments/__snapshots__/htmlComments.snap.twig

```diff
@@ -0,0 +1,14 @@
+<!-- I am a comment -->
+This is a paragraph
+
+<!-- I am a second comment -->
+
+Another paragraph
+
+<!-- I am a third comment -->
+
+A third paragraph
+
+<!--  This is the story of a little lamb 
+     that gets lost in a dark forest and
+     struggles to find home  -->
```

Comment:
Having extra whitespace after `<!--` and berfore `-->` on multiline html comment. Is this the expected output? I never write multiline comment. The old snapshot seems to be that way.

Reply:
I also thought this was a little odd, as well as some of the other tests.
I have gone for the approach of matching the current snapshots. But an alternate way would be that we have some tests that are skipped, and set up the snapshot to be a more "expected" output, which would give some good milestones for a v1 release.